app/controllers/application_drafts_controller.rb

```diff
@@ -34,9 +34,10 @@ def remove_question
 
   def update
     draft_params = params.require(:draft)
-    question_params = draft_params.require(:questions_attributes).permit!
+    question_attributes = draft_params.require(:questions_attributes)
+                                      .permit!.to_h
```

Comment:
Can you explain this one? I have a vague feeling that we're doing something questionable here.